microsoft-edge/webview2/how-to/debug-visual-studio-code.md

```diff
@@ -153,6 +154,34 @@ If you're debugging Office Add-ins, open the add-in source code in a separate in
 ```
 
 
+<!-- ====================================================================== -->
+## Debug WebView2 UWP Apps
```

Comment:
```suggestion
## Debug WebView2 WinUI 2 (UWP) Apps
```
@jasonstephen15 - nomenclature, maybe we want to name this fwk/plat. as "WinUI 2 (UWP)" rather than "UWP".

Get started with WebView2 in WinUI 2 (UWP) apps
https://learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2

WinUI 2 (UWP) sample app
https://learn.microsoft.com/en-us/microsoft-edge/webview2/samples/webview2_sample_uwp